### PR TITLE
Delta temp and atm stability docs

### LINKED ISSUE #89: docstring: atm_stb
child of #73

### LINKED ISSUE #90: docstring: delta_temp
child of #73

### PR DESCRIPTION
**Pull request by achaikou**
Fixes #89 
Fixes #90

### CODE REVIEW

File: camille/process/atm_stb.py [hunk 1]
```diff
@@ -3,12 +3,38 @@
...
     return atm_stb
 
 
 def _atm_stb(delta_temp):
+    """Atmospheric stability
+
+    Parameters
+    ----------
+    delta_temp : number
+        Temperature difference between measurments
```
Comment (JensGM) [inline camille/process/atm_stb.py]:
```suggestion
        Difference between ambient and sea temperatures
```

File: camille/process/atm_stb.py [hunk 2]
```diff
@@ -3,12 +3,38 @@
...
 
 def process(deltatemp):
+    """Process atmospheric stability
+
+    Verbally describe atmospheric stability
+
+    Parameters
+    ----------
+    deltatemp : pandas.Series
+        Temperature differences
```
Comment (JensGM) [inline camille/process/atm_stb.py]:
```suggestion
        Temperature difference
```

File: camille/process/atm_stb.py [hunk 3]
```diff
@@ -3,12 +3,38 @@
 
 
 def process(deltatemp):
+    """Process atmospheric stability
+
+    Verbally describe atmospheric stability
```
Comment (JensGM) [inline camille/process/atm_stb.py]:
```suggestion
    Classify atmospheric stability based on `deltatemp`
```

**Comment by achaikou** [comment]
OK, one has to know when to flee the battle. :white_flag:
Suggestions applied

**Pull request merged**